### DISCUSSION TITLE
Touchpad not working after sleep T480

### DISCUSSION THREAD

**Discussion by PeterSzobonya** [valnoxy/t480-oc#88] [Q&A]
I upgraded to Ventura and it works fine but touchpad stops working after I close the lid. Does anyone have any idea why this happens? I attach my efi below.
[EFI.zip](https://github.com/valnoxy/t480-oc/files/11151820/EFI.zip)

**Comment by wifihifi** [comment]
Hi Peter,  I did a fresh install of Ventura 13.3 and the trackpad works for me after waking.  I have a USB-C dock attached to the usb-c port closest to the front of the machine and have displayport software running my external display along with an extended Apple keyboard and a mouse attached.  Note the USB-C dock only functions when connected to the 2nd USB-C port - not sure why.  Also after sleep the dock is no longer recognized, I have to unplug it, move it to the first USB-C port then back again to the 2nd one for everything to come back.  I use a dock as it makes it easer to swap machines at my desk.  If I start using this T480 more I'll try out plugging everything in separately to see if wake from sleep works with an HDMI display and direct attached mouse and keyboard (USB-A).
